agr_indexer/src/main/java/org/alliancegenome/indexer/config/IndexerConfig.java

```diff
@@ -33,8 +21,8 @@ public enum IndexerConfig {
 	ParalogyIndexer("paralogy", GeneToGeneParalogyIndexer.class, 4, 5000, 5000, 8, 1),
 	PhenotypeAnnotationIndexer("phenotypeAnnotation", PhenotypeAnnotationCurationIndexer.class, 4, 1500, 1500, 2, 1),
 	ReleaseInfoIndexer("release", ReleaseInfoIndexer.class, 1, 1, 1, 1, 1),
-	GeneToGeneOrthologyIndexer("geneToGeneOrthology", GeneToGeneOrthologyIndexer.class, 4, 1500, 1500, 8, 1)
-	;
+	GeneToGeneOrthologyIndexer("geneToGeneOrthology", GeneToGeneOrthologyIndexer.class, 4, 1500, 1500, 8, 1),
+	GeneSummaryIndexer("geneSummary", GeneSummaryIndexer.class, 4, 2000, 20, 8, 1);
```

Comment:
You will want the 20 to be higher that is how ofter batches go to ES. Typically you want them to be the same.

Reply:
Yup.